src/main/java/seedu/address/logic/commands/location/EditLocationCommand.java

```diff
@@ -29,44 +27,43 @@
     public static final String COMMAND_WORD = "editLocation";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the location identified "
-            + "by the index number used in the displayed location list. "
+            + "by Id or the index number used in the displayed location list. "
             + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
+            + "Parameters: INDEX (must be a positive integer) or "
+            + PREFIX_LOCATION_ID + "ID "
             + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_ADDRESS + "ADDRESS] "
+            + "[" + PREFIX_ADDRESS + "ADDRESS] \n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_LOCATION_ID + "L123A "
+            + PREFIX_NAME + "National University of Singapore "
+            + PREFIX_ADDRESS + "21 Lower Kent Ridge Rd, Singapore 119077\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_NAME + "National University of Singapore "
-            + PREFIX_ADDRESS + "21 Lower Kent Ridge Rd, Singapore 119077";
+            + PREFIX_ADDRESS + "21 Lower Kent Ridge Rd, Singapore 119077";;
```

Comment:
There might be a typo here (extra semi-colon)

Reply:
Yeah it is. Have removed it.